packages/fxa-auth-server/scripts/paypal-processor.ts

```diff
@@ -48,7 +49,11 @@ export async function init() {
     database,
     senders.email
   );
+  const statsd = Container.get(StatsD);
+  statsd.increment('paypal-processor.startup');
   await processor.processInvoices();
+  statsd.increment('paypal-processor.shutdown');
+  await promisify(statsd.close).bind(statsd)();
```

Comment:
TIL: This is required to flush the `paypal-processor.shutdown` metric, otherwise it doesn't get logged.

Reply:
It's frequently not needed in longer running services where its not a big deal if a metric at the end doesn't make it out before the server shuts down. In this case we really want that metric.